Common.md

```diff
@@ -20,3 +42,140 @@ Language names are title cased.
 
 A full `User-Agent` example might look like `NewRelic-Python-TelemetrySDK/0.2.3 NewRelic-Python-OpenCensus/0.1.0`.
+
+## Attributes
+
+The origins of measurements exported to New Relic needs to be identifiable in order for them to be useful.
+Identify what generated, measured, collected, and transmitted measurements to New Relic is essential in providing the curated experience customers expect.
+This section describes the attributes that are used to serve this purpose.
+
+The following table is an overview for quick reference of all the attributes defined in the following sub-sections.
+
+| Attribute | REQUIRED/RECOMMENDED/OPTIONAL |
+| --------- | ----------------------------- |
+| `instrumentation.provider` | REQUIRED |
+| `instrumentation.name` | RECOMMENDED |
+| `instrumentation.version` | RECOMMENDED |
+| `collector.name` | RECOMMENDED |
+| `collector.version` | RECOMMENDED |
+| `host.hostname` | REQUIRED† |
+| `service.name` | REQUIRED† |
+| `app.name` | REQUIRED† |
+| `mobileApp.name` | REQUIRED† |
+| `device.model` | RECOMMENDED† |
+| `browserApp.name` | REQUIRED† |
+| `db.name` | REQUIRED† |
+
+†: applies only if annotating an applicable source.
+See the related sub-section for more information.
+
+### Instrumentation
+
+Instruments are used to make measurements of a computing system.
+The collective application of a set of instruments to comprehensively monitor a computing system is the instrumentation.
+
+For example:
+
+- Instrumentation for an HTTP library might be the dedicated code in the library that measure its performance (e.g. latency, throughput, errors).
+- Instrumentation for a cluster of Kubernetes pods might be the health check probes.
+- Instrumentation for a computer's CPU might be the embedded sensors measuring its physical state (e.g temperature, speed, power).
+
+Including attributes to identify what instrumentation is used is crucial for interoperability between telemetry systems and the New Relic platform.
+
+#### `instrumentation.provider`
+
+All exported measurements MUST contain an `instrumentation.provider` attribute that identifies the provider of instrumentation.
+
+Example values of this attribute are:
+
+* `newRelic`
+* `prometheus`
+* `dropwizard`
+* `opencensus`
+* `opentelemetry`
+
+#### `instrumentation.name`
+
+All exported measurements SHOULD contain an `instrumentation.name` attribute that identifies the code that is being instrumented if the instrumentation provider exposes this information, otherwise it SHOULD be omitted.
+This would be the framework, library, or application that uses the instrumentation to report measurements.
+
+Example values of this attribute are:
+
+* `spring-framework`
+* `python-twisted`
+* `gorilla/mux`
+
+#### `instrumentation.version`
+
+All exported measurements SHOULD contain an `instrumentation.version` attribute set to the version of the instrumentation if the instrumentation provider expose this information, otherwise it SHOULD be omitted.
```

Comment:
Add an additional sentence like:
`Note: For libraries, this is the version of the _instrumentation_, not the version of the library being instrumented.`

Reply:
Is it not already clearly stating that?

> ...set to the version of the instrumentation...

Reply:
To you and me, yes, but this distinction has caused a ton of confusion in OTel, so having an additional clarifying sentence seems like it could save some future questions. :)

Reply:
sounds good.